docs/spark-connector/spark-catalog-iceberg.md

```diff
@@ -97,44 +97,29 @@ DESC EXTENDED employee;
-| `jdbc-password`                 | `jdbc.password`                       | (none)        | Yes      | JDBC password             | 0.5.0         |
+| Gravitino catalog property name | Spark Iceberg connector configuration | Description                                                                                                                                                                                                         | Since Version |
+|---------------------------------|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| `catalog-backend`               | `type`                                | Catalog backend type                                                                                                                                                                                                | 0.5.0         |
+| `uri`                           | `uri`                                 | Catalog backend uri                                                                                                                                                                                                 | 0.5.0         |
+| `warehouse`                     | `warehouse`                           | Catalog backend warehouse                                                                                                                                                                                           | 0.5.0         |
+| `jdbc-user`                     | `jdbc.user`                           | JDBC user name                                                                                                                                                                                                      | 0.5.0         |
+| `jdbc-password`                 | `jdbc.password`                       | JDBC password                                                                                                                                                                                                       | 0.5.0         |
+| `io-impl`                       | `io-impl`                             | The io implementation for `FileIO` in Iceberg.                                                                                                                                                                      | 0.6.0         |
+| `s3-endpoint`                   | `s3.endpoint`                         | An alternative endpoint of the S3 service, This could be used for S3FileIO with any s3-compatible object storage service that has a different endpoint, or access a private S3 endpoint in a virtual private cloud. | 0.6.0         | 
```

Comment:
What's the meaning of `alternative`? Is this configuration optional?

Reply:
yes

Reply:
S3 had a global endpoint, and user doesn't need to set it by default